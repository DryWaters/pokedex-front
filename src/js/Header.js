import React from 'react';

import '../styles/Header.css';

class Header extends React.Component {
  render () {
    return (
      <div className="header" align="center">
        <h1> Pokedex </h1>
      </div>
    );
  }
}

export default Header;
