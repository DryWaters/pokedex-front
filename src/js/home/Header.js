import React from 'react';

import '../../styles/home/Header.css';

class Header extends React.Component {
  render () {
    return (
      <div className="header" align="center">
        <h1> Pok√©dex </h1>
      </div>
    );
  }
}

export default Header;
