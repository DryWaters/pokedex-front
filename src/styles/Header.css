.header {
  border-bottom: solid;
  margin: 8px;
}
