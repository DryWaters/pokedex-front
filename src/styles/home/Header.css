.header {
  border-bottom: solid;
  margin: 20px;
}
