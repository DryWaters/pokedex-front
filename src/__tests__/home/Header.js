import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../js/home/Header';

describe('Header Component', () => {
  it('should render Header component without crashing', () => {
    shallow(<Header />);
  });

  it('should match Header component snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
