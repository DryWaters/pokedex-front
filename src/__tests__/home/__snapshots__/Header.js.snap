// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should match Header component snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "align": "center",
      "children": <h1>
         Pokedex 
</h1>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Pokedex ",
      },
      "ref": null,
      "rendered": " Pokedex ",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "align": "center",
        "children": <h1>
           Pokedex 
</h1>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Pokedex ",
        },
        "ref": null,
        "rendered": " Pokedex ",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
